genesyscloud/telephony_provider_edges_trunkbasesettings/genesyscloud_telephony_providers_trunkbasesettings_schema.go

```diff
@@ -0,0 +1,114 @@
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "terraform-provider-genesyscloud/genesyscloud/resource_register"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+const (
+	resourceName = "genesyscloud_telephony_providers_edges_trunkbasesettings"
+)
+
+func SetRegistrar(l registrar.Registrar) {
+	l.RegisterDataSource("genesyscloud_telephony_providers_edges_trunkbasesettings", DataSourceTrunkBaseSettings())
+	l.RegisterResource("genesyscloud_telephony_providers_edges_trunkbasesettings", ResourceTrunkBaseSettings())
+	l.RegisterExporter("genesyscloud_telephony_providers_edges_trunkbasesettings", TrunkBaseSettingsExporter())
+
+}
+
+func ResourceTrunkBaseSettings() *schema.Resource {
+	return &schema.Resource{
+		Description: "Genesys Cloud Trunk Base Settings",
+
+		CreateContext: provider.CreateWithPooledClient(createTrunkBaseSettings),
+		ReadContext:   provider.ReadWithPooledClient(readTrunkBaseSettings),
+		UpdateContext: provider.UpdateWithPooledClient(updateTrunkBaseSettings),
+		DeleteContext: provider.DeleteWithPooledClient(deleteTrunkBaseSettings),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "The name of the entity.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"state": {
+				Description: "The resource's state.",
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+			},
+			"description": {
+				Description: "The resource's description.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"trunk_meta_base_id": {
+				Description: "The meta-base this trunk is based on.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+
+			"properties": {
+				Description:      "trunk base settings properties",
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
+				DiffSuppressFunc: util.SuppressEquivalentJsonDiffs,
+			},
+			"trunk_type": {
+				Description:  "The type of this trunk base.Valid values: EXTERNAL, PHONE, EDGE.",
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice([]string{"EXTERNAL", "PHONE", "EDGE"}, false),
+			},
+			"managed": {
+				Description: "Is this trunk being managed remotely. This property is synchronized with the managed property of the Edge Group to which it is assigned.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+			"inbound_site_id": {
+				Description: "The site to which inbound calls will be routed. Only valid for External BYOC Trunks.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"site_id": {
```

Comment:
This is the new field being added

Reply:
I'm curious as to how it's different from inbound_site_id? What is the value supposed to be?

Reply:
So, just looking at my own test orgs, it seems like the site reference is an actual `genesyscloud_telephony_providers_edges_site` resource. For example:
```
❯ gc telephony providers edges trunkbasesettings list -p dg13 -a | grep site -A10
    "site": {
      "id": "af4ab1f0-3628-4a06-ac98-9584b0ebdff8",
      "name": "PureCloud Voice - AWS",
      "selfUri": "/api/v2/telephony/providers/edges/sites/af4ab1f0-3628-4a06-ac98-9584b0ebdff8"
    },
```
If it is a `genesyscloud_telephony_providers_edges_site` resource, can we update the docs with that in the example?